src/GraphControl/Control/Grapher.h

```diff
@@ -148,6 +148,104 @@ public
         Platform::String ^ ConvertToLinear(Platform::String ^ mmlString);
         void PlotGraph();
         void AnalyzeEquation(GraphControl::Equation ^ equation);
+       // We can't use the EValTrigUnitMode enum directly in as the property type because it comes from another module which doesn't expose
+        // it as a public enum class.  So the compiler doesn't recognize it as a valid type for the ABI boundary.
+        property int TrigUnitMode
+        {
+            void set(int value)
+            {
+                if (value != (int)m_solver->EvalOptions().GetTrigUnitMode())
+                {
+                    m_solver->EvalOptions().SetTrigUnitMode((Graphing::EvalTrigUnitMode)value);
+                    UpdateGraph();
+                }
+            }
+
+            int get()
+            {
+                return (int)m_solver->EvalOptions().GetTrigUnitMode();
+            }
+        }
+
+        property double XAxisMin
+        {
+            double get()
+            {
+                return m_graph->GetOptions().GetDefaultXRange().first;
```

Comment:
Here and the rest of these properties, it is possible that m_graph is null at various points, in the cpp file we often check for this, I think we should add some checks here also and return 0.0 if it is null or some other default value.